admin/tests/acceptance/user-details-personal-information-test.js

```diff
@@ -0,0 +1,48 @@
+import { module, test } from 'qunit';
+import { click, currentURL, fillIn, visit } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import { createAuthenticateSession } from 'pix-admin/tests/helpers/test-init';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+
+module('Acceptance | user details personal information', function(hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+  let user = null;
+
+  hooks.beforeEach(async function() {
+    await createAuthenticateSession({ userId: 2 });
+    user = this.server.create('user');
+  });
+
+  test('visiting /users/:id', async function(assert) {
+
+    // when
+    await visit(`/users/${user.id}`);
+
+    // then
+    assert.equal(currentURL(), `/users/${user.id}`);
+  });
+
+  module('edit users details', function() {
```

Comment:
Je pense que c'est mieux de privilégier les module avec des when : `module('when admin edit users details', () {`

Reply:
ça fait pas mal de fois qu'on remonte cette question: quand est-ce qu'on fait des module, des context, when ou en fin de it. Ca vous dirait pas qu'on décide qqch, quitte à changer après ?

Reply:
Done